build.rs

```diff
@@ -72,10 +72,10 @@ fn main() {
     {
         let emoji_vs_list = Path::new(&out).join("emoji-variation-sequences.txt");
         if !std::fs::read_to_string(&emoji_vs_list)
-            .is_ok_and(|text| text.contains("Emoji Version 16.0"))
+            .is_ok_and(|text| text.contains("Emoji Version 17.0"))
```

Comment:
```suggestion
            .is_ok_and(|text| text.contains("Version: 17.0"))
```

This line is not present is the new version of the file so it needs to be changed.